pkg/sbproxy/reconcile/reconcile_brokers_test.go

```diff
@@ -373,6 +374,46 @@ var _ = Describe("Reconcile brokers", func() {
 				}
 			},
 		}),
+
+		Entry("When broker is registered in the platform, but not yet known to the proxy, it should be updated", testCase{
+			stubs: func() {
+				stubPlatformOpsToSucceed()
+				fakePlatformBrokerClient.UpdateBrokerReturns(&platform.ServiceBroker{
+					GUID:      platformbrokerNonProxy.GUID,
+					Name:      "sm-proxy-smBrokerID",
+					BrokerURL: fakeAppHost + "/smBrokerID",
+				}, nil)
+			},
+			platformBrokers: func() ([]platform.ServiceBroker, error) {
+				return []platform.ServiceBroker{
+					platformbrokerNonProxy,
+				}, nil
+			},
+			smBrokers: func() ([]sm.Broker, error) {
+				return []sm.Broker{
+					sm.Broker{
+						ID:               "smBrokerID",
+						BrokerURL:        platformbrokerNonProxy.BrokerURL,
+						Metadata:         platformbrokerNonProxy.Metadata,
+						ServiceOfferings: platformbrokerNonProxy.ServiceOfferings,
+					},
+				}, nil
+			},
+			expectations: func() expectations {
+				return expectations{
+					reconcileCreateCalledFor:  []platform.ServiceBroker{},
+					reconcileDeleteCalledFor:  []platform.ServiceBroker{},
+					reconcileCatalogCalledFor: []platform.ServiceBroker{},
+					reconcileUpdateCalledFor: []platform.ServiceBroker{
+						platform.ServiceBroker{
+							GUID:      platformbrokerNonProxy.GUID,
```

Comment:
could you (similarly like other tests use just a variable with a meaningful name in the expectations (keep it readable)